lib/assertions.ts

```diff
@@ -955,6 +955,51 @@ export default class DOMAssertions {
     }
   }
 
+  /**
+   * Assert that the tagName of the {@link HTMLElement} or an {@link HTMLElement}
+   * matching the `selector` matches the `expected` tagName, using the
+   * [`tagName`](https://developer.mozilla.org/en-US/docs/Web/API/Element/tagName)
+   * property of the {@link HTMLElement}.
+   *
+   * @param {string} expected
+   * @param {string?} message
+   *
+   * @example
+   * // <h1 id="title">
+   * //   Title
+   * // </h1>
+   *
+   * assert.dom('#title').hasTag('h1');
+   */
+  hasTag(tagName: string, message?: string) {
```

Comment:
I think I would prefer `hasTagName()` 🤔 

thoughts?

Reply:
Ha, I quite literally thought the same thing. I'll make the change now.